ansible/roles/monitoring-influxdb/templates/influxdb.users.sh.j2

```diff
@@ -2,6 +2,18 @@ influx -database "metrics_db" -execute "create user {{ influx_admin_user }} with
 
 influx -database "metrics_db" -username {{ influx_admin_user }} -password {{ influx_admin_password }} -execute "create user {{ alertd_influx_user }} with password '{{ alertd_influx_pass }}'"
 influx -database "metrics_db" -username {{ influx_admin_user }} -password {{ influx_admin_password }} -execute "grant read on metrics_db to {{ alertd_influx_user }}"
-influx -database "collectd_db" -username {{ influx_admin_user }} -password {{ influx_admin_password }} -execute "grant read on collectd_db to {{ alertd_influx_user }}"  
+influx -database "collectd_db" -username {{ influx_admin_user }} -password {{ influx_admin_password }} -execute "grant read on collectd_db to {{ alertd_influx_user }}"
```

Comment:
Used a for loop with jinja2 variables for the databases instead of manually listing every single db.